build/configs/taste.yara

```diff
@@ -224,6 +224,17 @@ rule olecf_file {
         uint32(0) == 0xE011CFD0 and uint32(4) == 0xE11AB1A1
 }
 
+rule onenote_file {
+    meta:
+        author = "Aiden Mitchell"
+        description = "Microsoft Onenote File"
+        type = "document"
+    strings:
+        $a = { E4 52 5C 7B 8C D8 A7 4D AE B1 53 78 D0 29 96 D3 }
```

Comment:
Could you add a comment with what this translates to? I think most or all of the `strings` matches in here have that, e.g. line 213.